sdk/error.go

```diff
@@ -408,8 +408,17 @@ func (e Error) Error() string {
 		message = errorsAmericanEnglish[ErrUnknownError.ID]
 	}
 	if e.From != "" {
-		message = fmt.Sprintf("%s (from: %s)", message, e.From)
+		if e.RequestID != "" {
```

Comment:
Maybe you can add/edit test cases for the request id print in error_test.go

Reply:
done